### ISSUE TITLE
Proxypass stops working on 0.33

### ISSUE THREAD

**Issue by danaivehr** [metabase/metabase#10657]
**Describe the bug**
After upgrade from 0.32.3 to 0.33 proxypass https:// to http://localhost stops working

**Logs**
Seems like metabase just don't react on connection at all - no messages in logs about access

**To Reproduce**
1. Run metabase.jar on localhost:3000
2. Setup proxypass on apache2:
```
    ProxyPass /metabase/ http://localhost:3000/
    ProxyPassReverse /metabase/ http://localhost:3000/
```
3. Connect locally to http://localhost:3000 - can get to login screen
4. Connect to https://myserver.com/metabase - Page not found + "GET /metabase/ HTTP/1.1" 404 459 in logs
5. Replace metabase 0.33 with 0.32.3 -- can get to login screen

**Expected behavior**
Can get to login screen through https://myserver.com/metabase

**Information about your Metabase Installation:**
- Debian
- Apache2 with proxypass ssl https:// to plain http://localhost:3000
- Postgres
- Metabase.jar version 0.32 - works OK
- Metabase.jar version 0.33 - has this issue

It is obvious that something is changed since 0.32.3, but i can not figure it out.

**Comment by danaivehr** [comment]
UPDATE: 
I found out, that metabase web page is loading, but blank.

If i go to https://myserver.com/metabase 

it seems like empty page, but it is not empty:

https://pastebin.com/stYCgRSz 

If i use https://localhost:3000 logs say

`09-11 15:19:47 DEBUG middleware.log :: GET /api/session/properties 200 63.4 ms (1 DB calls) Jetty threads: 6/50 (2 idle, 0 queued) (67 total active threads) Queries in flight: 0
09-11 15:19:47 DEBUG middleware.log :: GET /api/user/current 200 64.3 ms (3 DB calls) Jetty threads: 6/50 (2 idle, 0 queued) (67 total active threads) Queries in flight: 0
09-11 15:19:47 DEBUG middleware.log :: GET /api/user/current 200 2.3 ms (3 DB calls) Jetty threads: 3/50 (4 idle, 0 queued) (68 total active threads) Queries in flight: 0
09-11 15:19:47 DEBUG middleware.log :: GET /api/database 200 4.2 ms (3 DB calls) Jetty threads: 3/50 (3 idle, 0 queued) (68 total active threads) Queries in flight: 0
09-11 15:19:47 DEBUG middleware.log :: GET /api/collection/root/items 200 6.5 ms (7 DB calls) Jetty threads: 3/50 (4 idle, 0 queued) (68 total active threads) Queries in flight: 0
09-11 15:19:47 DEBUG middleware.log :: GET /api/setting 200 903.0 µs (0 DB calls) Jetty threads: 3/50 (4 idle, 0 queued) (68 total active threads) Queries in flight: 0
09-11 15:19:48 DEBUG middleware.log :: GET /api/collection/root 200 2.3 ms (2 DB calls) Jetty threads: 3/50 (4 idle, 0 queued) (68 total active threads) Queries in flight: 0
09-11 15:19:48 DEBUG middleware.log :: GET /api/database 200 3.6 ms (3 DB calls) Jetty threads: 3/50 (4 idle, 0 queued) (68 total active threads) Queries in flight: 0
09-11 15:19:48 DEBUG middleware.log :: GET /api/automagic-dashboards/database/2/candidates 200 76.4 ms (7 DB calls) Jetty threads: 3/50 (4 idle, 0 queued) (68 total active threads) Queries in flight: 0
`

If i use https://myserver.com/metabase **log says nothing**

What can i do to investigate this issue?

**Comment by salsakran** [comment]
Hello @danaivehr  -

The logs don't have anything strange, it all looks clear. Given that we don't explicitly support proxy pass, and that overall there isn't something specific on our end that's broken, I'm closing this.

For questions around how to configure Metabase with other software, I suggest asking on our user forum - discourse.metabase.com

If you think there is a specific bug, please share the bug report and we'll re-open this.

**Comment by northanapon** [comment]
I know you closed this issue but I have the exact same issue with Apache2 ProxyPass since the update 0.33. When accessing the Metabase with non-root URL such as http://sampledurl.com/metabase/, the console will show a bunch of 404 errors such as http://sampledurl.com/app/dist/app-main.bundle.css?4ccd5c23fd3dcf61a206fe52be657d90.

<img width="1099" alt="Screen Shot 2019-10-24 at 3 32 00 PM" src="https://user-images.githubusercontent.com/639215/67467723-7bc9b780-f673-11e9-9643-381da2c1897a.png">

**Comment by flamber** [comment]
@northanapon There's a regression in 0.33.4, which will be fixed by #11206, and most likely be in 0.33.5. The only solution currently is to downgrade to 0.33.3 or create a handful of rewrite rules.

**Comment by danaivehr** [comment]
> I know you closed this issue but I have the exact same issue with Apache2 ProxyPass since the update 0.33.4. When accessing the Metabase with a non-root URL such as http://sampledurl.com/metabase/, the console will show a bunch of 404 errors such as http://sampledurl.com/app/dist/app-main.bundle.css?4ccd5c23fd3dcf61a206fe52be657d90.

I can confirm the same behavior - 404 for some files load with wrong url without prefix.
I played around with some rewrite rules, but i'm not good in it, to make a work-around. Still stacked.

**Comment by mazameli** [comment]
@paulrosenzweig believes this should actually be fixed in 0.33.5.1 via https://github.com/metabase/metabase/pull/11174. We neglected to include this fix in the release notes.
